test/plugin/containers/jvm-container/pom.xml

```diff
@@ -32,95 +32,44 @@
-                                    <goal>build</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-
-        <profile>
-            <id>jdk14</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>io.fabric8</groupId>
-                        <artifactId>docker-maven-plugin</artifactId>
-                        <version>0.33.0</version>
-                        <configuration>
-                            <images>
-                                <image>
-                                    <name>skywalking/agent-test-jvm:jdk14-${project.version}</name>
-                                    <build>
-                                        <from>openjdk:14-buster</from>
-                                        <workdir>/usr/local/skywalking/scenario</workdir>
-                                        <assembly>
-                                            <mode>dir</mode>
-                                            <targetDir>/usr/local/skywalking</targetDir>
-                                            <descriptor>assembly.xml</descriptor>
-                                        </assembly>
-                                        <runCmds>
-                                            <run>chmod +x /usr/local/skywalking/run.sh</run>
-                                            <run>tar -xvf ../tools/skywalking-mock-collector.tar.gz -C ../tools</run>
-                                        </runCmds>
-                                        <cmd>["/usr/local/skywalking/run.sh"]</cmd>
-                                    </build>
-                                </image>
-                            </images>
-                        </configuration>
-                        <executions>
-                            <execution>
-                                <id>build</id>
-                                <phase>package</phase>
-                                <goals>
-                                    <goal>build</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.33.0</version>
+                <configuration>
+                    <images>
+                        <image>
+                            <name>skywalking/agent-test-jvm:jdk${java_major_version}-${project.version}</name>
+                            <build>
+                                <from>openjdk:${java_major_version}-buster</from>
```

Comment:
Same here.

Reply:
Please use the same base images as what we are using in our official Docker images

https://github.com/apache/skywalking-java/blob/02d6b9c2d85da2b1b1ce3e2a8013373e0e371ba1/Makefile#L44-L51

Reply:
@dmsolr told me there are some error in script in alpine, and our previous testing is on the top of this version due to the limitation of our scripts

Reply:
![image](https://user-images.githubusercontent.com/5441976/140612141-401d1720-fa9c-4a33-ae19-f0b4226508a0.png)

I think we were using this. We tried openjdk:alpine here IIRC, the tests would fail